Makefile

```
@@ -6,8 +6,13 @@ TARGET_DIR=bin/
 
 .PHONY: build test clean
 
+fmt:
+	goimports -w cmd/*/*.go pkg/*/*.go
+	gofmt -l -w .
```

gofmtが入っていない環境を考慮したいので，goコマンドにバンドルされている`go fmt`を使う．

```suggestion
	go fmt ./...
```